lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -69,19 +92,11 @@ def handle(self, *args, **options):
 
         if options['add'] or options['del']:
             user_str = options['add'] or options['del']
-            if '@' in user_str:
-                user = User.objects.get(email=user_str)
-            else:
-                user = User.objects.get(username=user_str)
-
-            cert_whitelist, _created = \
-                CertificateWhitelist.objects.get_or_create(
-                    user=user, course_id=course)
-            if options['add']:
-                cert_whitelist.whitelist = True
-            elif options['del']:
-                cert_whitelist.whitelist = False
-            cert_whitelist.save()
+            add_to_whitelist = True if options['add'] else False
+            users_list = user_str.split(",")
+            for username in users_list:
+                if username:
```

Comment:
you might want to change this to `if username.strip()` in case there's a string with blank spaces